src/main/kotlin/org/malachite/estella/queues/ExampleConsumer.kt

```diff
@@ -0,0 +1,33 @@
+package org.malachite.estella.queues
+
+import com.rabbitmq.client.AMQP
+import com.rabbitmq.client.DefaultConsumer
+import com.rabbitmq.client.Envelope
+
+class ExampleConsumer() : Queue() {
+
+    var received = 0
+
+    init {
+        channel.queueDeclare("hello", false, false, false, null)
+        println(" [*] Waiting for messages. To exit press CTRL+C")
+        val consumer = object : DefaultConsumer(channel) {
+            override fun handleDelivery(
+                consumerTag: String,
+                envelope: Envelope,
+                properties: AMQP.BasicProperties,
+                body: ByteArray
+            ) {
+                val message = String(body, charset("UTF-8"))
+                received+=1
+                println(" [x] Received '$message'")
+            }
+        }
+        channel.basicConsume("hello", true, consumer)
+    }
+
+    fun stopConsumer(){
+        channel.queueDelete("hello")
```

Comment:
Pytanie czy trzeba usuwać te kolejkę? Wiem, że to tylko example (ale Ty mi się też do examplow przypierdzielałes XDDD - zemsta XDDD), ale ogólnie my raczej tych kolejek nie będziemy chcieli usuwać.

Reply:
Ponadto to chyba takie ostre - żeby zatrzymać konsumenta to usuwasz kolejkę. Trochę ostro siostro. Jestem pewny, że da się go jakoś inaczej zatrzymać, a nie wiem, czy Rabbit tego jakoś ostro nie traktuje.

Reply:
Znaczy trochę mam gdzieś jak to rabbit traktuje, ale ok jak uważasz że trzeba XDD Plus bez usuwania kolejki o ile pamiętam to elementy się dołączające będą czytały z niej stare wiadomości chyba??